spt/OrangeBox/overlay/portal_camera.cpp

```diff
@@ -0,0 +1,195 @@
+#include "stdafx.h"
+
+#ifdef SSDK2007
+
+#include "portal_camera.hpp"
+#include "..\modules.hpp"
+#include "..\spt-serverplugin.hpp"
+#include "cdll_int.h"
+#include "engine\iserverplugin.h"
+#include "eiface.h"
+#include "tier2\tier2.h"
+#include "edict.h"
+#include "..\cvars.hpp"
+#include "mathlib\vmatrix.h"
+
+
+const int PORTAL_ORIGIN_OFFSET = 1180;
+const int PORTAL_ANGLE_OFFSET = 1192;
+const int PORTAL_LINKED_OFFSET = 1068;
+const int INDEX_MASK = MAX_EDICTS - 1;
+
+
+bool get_portal_index(edict_t ** portal_edict, Vector & new_player_origin, QAngle & new_player_angles)
```

Comment:
Fix the whitespace around pointers and references please here and elsewhere in the file.